documentation/asciidoc/topics/proc_configuring_xsite_within_clusters.adoc

```diff
@@ -38,9 +38,12 @@ spec:
 +
 . Configure nodes with any other {datagridservice} resources.
 . Apply the `Infinispan` CRs.
-. Check node logs to verify that {brandname} clusters form a cross-site view, for example:
+. Verify that {brandname} clusters form a cross-site view.
+.. Retrieve the `Infinispan` CR.
 +
 [source,options="nowrap",subs=attributes+]
 ----
-include::cmd_examples/oc_grep_xsite.adoc[]
```

Comment:
@oraNod do we need to remove `oc_grep_xsite.adoc` from the documentation?

Reply:
Yes, please.